web/src/lib/components/shared-components/autogrow-textarea.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+  import { autoGrowHeight } from '$lib/actions/autogrow';
+  import { clickOutside } from '$lib/actions/click-outside';
+  import { shortcut } from '$lib/actions/shortcut';
+  import { tick } from 'svelte';
+
+  export let content: string = '';
+  export let elementClass: string = '';
```

Comment:
This is more consistent with other components:

```suggestion
  let elementClass: string = '';
  export { elementClass as class };
```